deeposlandia/network.py

```diff
@@ -5,20 +5,32 @@
 
 class ConvolutionalNeuralNetwork:
     """Convolutional neural network design
+
+    Attributes
+    ----------
+    _network_name : str
```

Comment:
It's quite strange for me to have some "protected" attributes in the docstring. The most of the time, you write `self._attr` when you don't want to let it be "public", so you don't write a docstring for it.

I prefer to have "public" and explicit attributes which define you typed object. Is it possible to replace the `self._attr` by `self.attr`?

Reply:
Yes, these stuffs are remainders of a quite clumsy C++-convention adaptation. Useless in Python, I remove them.